custom_components/tuya_local/devices/doel_tiplus_litterbox.yaml

```diff
@@ -0,0 +1,249 @@
+name: Litter Box
+products:
+  - id: ebf44398a24b7b6be3vnml
+    name: DOEL Ti+
+primary_entity:
+  entity: sensor
+  name: Cat Weight
+  icon: "mdi:cat"
+  class: weight
+  dps:
+    - id: 6
+      type: integer
+      name: sensor
+      unit: kg
+      class: measurement
+      mapping:
+        - scale: 10
+    - id: 134
+      type: integer
+      name: sensor
```

Comment:
Only one dp within the entity can have the name sensor.  The other will need to be renamed and will be an extra attribute (in which case, unit will not be used, so make it clear in the name).

If the intention is to have this dependant on the unit selection, you need to use:
```
    - id: 6
      type: integer
      name: sensor
      class: measurement
      mapping:
        - scale: 10
          constraint: unit
          conditions:
            - dps_val: lb
              value_redirect: weight_lb
    - id: 108
      type: string
      name: unit
    - id: 134
      type: integer
      name: weight_lb
```
I don't remember whether the scale from the original mapping applies to the redirected value, so you may need to add the mapping with `scale:10` under weight_lb as well.